auxiliaries.py

```diff
@@ -81,6 +81,11 @@ def vprint(level, *args, **kwargs):
 
     class_instance.vprint = vprint
 
+def get_data(base_path="/home/pi/data-log"):
```

Comment:
You should import `DEFAULT_DATALOG` from `global_values.py` and use that for the default here.